srcpkgs/buildifier/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'buildifier'
+pkgname=buildifier
+version=5.1.0
+revision=1
+wrksrc="buildtools-${version}"
+build_style=go
+go_import_path="github.com/bazelbuild/buildtools/buildifier"
+short_desc="Format bazel BUILD and .bzl files with a standard convention"
+maintainer="n1c00o <git.n1c00o@gmail.com>"
+license="Apache-2.0"
+homepage="https://github.com/bazelbuild/buildtools/tree/master/buildifier"
+#changelog="https://github.com/bazelbuild/buildtools/releases"
+distfiles="https://github.com/bazelbuild/buildtools/archive/refs/tags/${version}.tar.gz"
+checksum=e3bb0dc8b0274ea1aca75f1f8c0c835adbe589708ea89bf698069d0790701ea3
+tags="bazel build format lint"
```

Comment:
`tags` is seldomly used in Voidlinux packages and its usefulness is questionable. `xbps-query` can't search packages by tags, so it only appears in the package metadata and it isn't directly used by XBPS nor any utility that I would know of.

```sh
. /usr/share/xbps.d/00-repository-main.conf && wget -qO - "$repository/x86_64-repodata" | tar --zstd -xOf - index.plist | fgrep -B 40 "<key>tags"
```
This shell command searches for everything in the repository index that contains the `tags` property. This shows that the only package in the main repository with the `x86_64` architecture is `tuigreet`. This shows that only a single package in the entire Voidlinux repository is using `tags`.